indico/modules/events/registration/templates/display/registration_modify.html

```diff
@@ -16,6 +16,8 @@
 
 {% block content %}
     {% if registration.can_be_modified or registration.invitation %}
+        {{ template_hook('before-regform', event=event, regform=registration.registration_form,
+                         registration=registration, management=False) }}
```

Comment:
lowercase `false` (can you also fix it in the existing hook below)?

(jinja recommends lowercase true/false/none even though the python-style capitalization works as well)

Reply:
Good, I just copy and paste from the line below for `after-regform`
. I'm going to fix also the line below.